src/components/apiTokens/APITokenList.tsx

```diff
@@ -11,6 +11,15 @@ import DeleteAPITokenModal from './DeleteAPITokenModal'
 import NoResults from '../../assets/img/empty-noresult@2x.png'
 import './apiToken.scss'
 import EmptyState from '../EmptyState/EmptyState'
+import { ReactComponent as Question } from '../../assets/icons/ic-help-outline.svg'
+import { ReactComponent as QuestionFilled } from '../../assets/icons/ic-help-filled.svg'
+import TippyCustomized from '../common/TippyCustomized'
+
+
+enum TippyTheme {
```

Comment:
We have classes for tippy for both black & white. Verify that the theme & classes have same attributes

Reply:
done